component/componenttest/nop_exporter_test.go

```diff
@@ -33,19 +33,19 @@ func TestNewNopExporterFactory(t *testing.T) {
 	cfg := factory.CreateDefaultConfig()
 	assert.Equal(t, &nopExporterConfig{ExporterSettings: config.NewExporterSettings(config.NewID("nop"))}, cfg)
 
-	traces, err := factory.CreateTracesExporter(context.Background(), component.ExporterCreateParams{}, cfg)
+	traces, err := factory.CreateTracesExporter(context.Background(), component.ComponentSettings{}, cfg)
```

Comment:
Since this is used a lot in tests (lot's of time with zap.Nop), probably good to have in `componenttest` a `NopComponentSettings` or `TestComponentSettings` that uses zap.Nop + DefaultBuildInfo and pass that in tests.